examples/sdp3x_manual_address/sdp3x_manual_address.ino

```diff
@@ -1,7 +1,7 @@
-#include <Wire.h>
-
 #include <sdpsensor.h>
 
+#include <Wire.h>
```

Comment:
Is there a specific reason to change the include order here? Otherwise I'd leave as much as possible unchanged.